tests/integration/network_test.go

```diff
@@ -69,8 +63,6 @@ func (s *NetworkTestSuite) TestGetLanes() {
 func (s *NetworkTestSuite) TestGetAuctionParams() {
 	s.T().Parallel()
```

Comment:
Do we run into race-conditions here? If all tests are performed against the same network in parallel?

Reply:
Simple query tests can be run in parallel, but if you want to do things sequentially, you can just not include this `Parallel()` call for your specific test